test/data/build.yaml

```diff
@@ -1,3 +1,16 @@
 # Build Example
+id: 8843d7f92416211de9ebb963ff4ce28125932878
 jobId: 50dc14f719cdc2c9cb1fb0e49dd2acc4cf6189a0
-runNumber: 15
+parentBuildId: d8fd39d0bbdd2dcf322d8b11390a4c5825b11495
+number: 15
```

Comment:
so would the build number be `d8fd39d0bbdd2dcf322d8b11390a4c5825b11495.15` in this case?